app/controllers/bookings_controller.rb

```diff
@@ -1,2 +1,41 @@
 class BookingsController < ApplicationController
+  before_action :find_dress
```

Comment:
We'll have to add an only: [:new, :edit] when we comment the destroy method back in since we don't we don't want to find any dress ids when we go for deleting bookmarks.

Reply:
Got it 👍, I had the only in before commenting destroy out and got rid of it. I’ll put those back in